README.md

```diff
@@ -1 +1,19 @@
-# data-structures
\ No newline at end of file
+`   # data-structures
+
+Got help from Justin(github/welliam) for to deal with iterable. 
+Data Structure Solutions for Code Fellows Python Class
+
+1. Singlely Linked List
+push, size, __len__, search, pop, remove, display
+
+2. Stack
+push, pop
+
+3. Doubly Linked List
+push, append, pop, shift, remove
+
+4. Queue
+enqueue, dequeue, peek, size
+
+5. Deque
+append, appendleft, pop, popleft, peek, peekleft, size
```

Comment:
For each of these data structures, you should provide a short description of the methods used. One sentence per public method (so not including `__len__`) should do the trick.